crates/bevy_winit/src/lib.rs

```diff
@@ -344,12 +346,26 @@ fn handle_winit_event(
     redraw_event_reader: &mut ManualEventReader<RequestRedraw>,
     winit_events: &mut Vec<WinitEvent>,
     exit_notify: &SyncSender<AppExit>,
+    exit_happened: &mut Option<AppExit>,
     event: Event<UserEvent>,
     event_loop: &EventLoopWindowTarget<UserEvent>,
 ) {
     #[cfg(feature = "trace")]
     let _span = bevy_utils::tracing::info_span!("winit event_handler").entered();
 
+    // We guarantee that after a user send a `AppExit` the game won't update again.
+    // Problem is winit will send us queued events even after we have triggered a
+    // loop exit. These events then trigger a game update.
+    //
+    // TODO: This is a bodge. A better solution would be to go and figure out
+    //       which winit events we need to handle after the app exited and letting
+    //       those through. The non-send PR will be replacing all of this so
```

Comment:
```suggestion
    //       those through. The non-send PR (#9122) will be replacing all of this so
```

I think we should link to the PR, since there's a chance that "non-send PR" could refer to multiple PRs in the future.